src/Uno.Toolkit.UI/Controls/LoadingView/LoadingView.cs

```diff
@@ -137,5 +137,24 @@ private void UpdateVisualState()
 
 			VisualStateManager.GoToState(this, loadingState, IsLoaded);
 		}
+
+		#region DependencyProperty: Source
```

Comment:
Should be `SplashScreen` instead of `Source`

Reply:
Splash screen property has been removed